server/lib/payments.js

```diff
@@ -428,8 +429,10 @@ const sendOrderConfirmedEmail = async (order, transaction) => {
       subscriptionsLink: interval && `${config.host.website}/${fromCollective.slug}/recurring-contributions`,
     };
 
-    // hit PDF service and get PDF
-    pdf = await getTransactionPdf(transaction, user);
+    // hit PDF service and get PDF (unless payment method type is gift card)
+    if (paymentMethod.type !== PAYMENT_METHOD_TYPES.VIRTUALCARD) {
```

Comment:
`paymentMethod` can be null (ie. if order amount is `0`).

```suggestion
    if (paymentMethod?.type !== PAYMENT_METHOD_TYPES.VIRTUALCARD) {
```